community/meta-frage.md

```diff
@@ -2,34 +2,32 @@
 tag: meta-frage
 alias: []
 ---
+## Metafragen
 
-```
-Metafragen
-Eine Metafrage ist eine Frage über eine Frage, wie beispielsweise „Darf ich etwas fragen?“ oder „Kennt sich jemand mit Computern aus?“.
+Metafragen sind Fragen über Fragen, wie "Darf ich etwas fragen?" oder "Kennt sich jemand mit Computern aus?". Oft suchen Hilfesuchende so nach kompetenten Ansprechpartnern. Diese Fragen werden oft ignoriert oder der Fragende wird darauf hingewiesen, dass eine konkrete Problembeschreibung nötig ist. In der Regel folgt auf eine Meta-Frage eine weitere Frage.
 
-In der Regel wird der Begriff Metafrage aber verallgemeinert und damit alle Fragen bezeichnet, die keine direkte Frage zum Problem des Hilfesuchenden sind. Der Hilfesuchende fragt also zunächst allgemein, ob jemand helfen kann. Gerade Neulinge oder unerfahrene Benutzer lassen sich zu Metafragen hinreißen, um einen kompetenten und hilfsbereiten Ansprechpartner zu finden. Meistens werden Metafragen ignoriert oder der Fragende wird rüde darauf hingewiesen, dass ihm niemand bei seinem Problem helfen könne, ohne dies zu kennen. Grundsätzlich folgt auf eine Meta-Frage eine weitere Frage.
+### Vorteile von Metafragen
 
+- Als höfliche Floskel um Aufmerksamkeit zu gewinnen
+- Beginn einer zunächst einseitigen Konversation (Allgemeine Problemanalyse)
 
-Vorteile von Metafragen
-als höfliche Floskel um Aufmerksamkeit zu gewinnen
-Beginn einer zunächst einseitigen Konversation (Allgemeine Problemanalyse)
+### Nachteile von Metafragen
 
-Nachteile von Metafragen
-die Anwesenden könnten eventuell bei dem Problem helfen, obwohl sie (eventuell aus Bescheidenheit) nicht von sich behaupten würden, mit dem Thema vertraut zu sein,
-oft ist die Metafrage falsch formuliert, z. B. wird gefragt „kennt sich jemand mit Kochen aus?“ und er will nur wissen, ob Mangos essbar sind,
-Auch wenn jemand mit dem erfragten Thema vertraut ist, bedeutet dies nicht, dass er eine spezielle Frage zu diesem beantworten kann – niemand ist allwissend,
-wenn keine Reaktion auf die Metafrage erfolgt, beläßt es der Fragende meist dabei. So kann eine Antwort auf das Problem von später aufmerksam Werdenden nicht erfolgen.
+- Anwesende könnten helfen, auch wenn sie sich möglicherweise nicht als Experten outen.
+- Metafragen sind oft ungenau formuliert, wie z. B. "Kennt sich jemand mit Kochen aus?", wenn es eigentlich um die Essbarkeit von Mangos geht.
+- Vertrautsein mit einem Thema bedeutet nicht, dass alle spezifischen Fragen dazu beantwortet werden können.
+- Fehlt eine Reaktion auf die Metafrage, bleibt der Fragende oft schweigend, was spätere Hilfe erschwert.
```

Comment:
Würde die Reihenfolge der Punkte leicht abändern, der Erste ist meiner Meinung nach nicht so relevant

```suggestion
- Metafragen sind oft ungenau formuliert, wie z. B. "Kennt sich jemand mit Kochen aus?", wenn es eigentlich um die Essbarkeit von Mangos geht.
- Vertrautsein mit einem Thema bedeutet nicht, dass alle spezifischen Fragen dazu beantwortet werden können.
- Anwesende könnten helfen, auch wenn sie sich möglicherweise nicht als Experten bezeichnen würden.
- Fehlt eine Antwort auf die Metafrage, wird das Problem oft entweder garnicht oder nur deutlich später beantwortet.
```

Habe außerdem die Formulierung des letzten Punkts leicht abgeändert, weil das Passiv den Satz schwieriger gemacht hat als er sein muss. Habe den letzten Teilsatz persönlich auch nicht wirklich verstanden.

Man könnte auch darüber nachdenken, ein IT-näheres Beispiel beim neuen ersten Punkt zu wählen. Mangos und Kochen haben ohnehin nicht besonders viel miteinander zu tun.

Reply:
Nehme gerne vorschläge an :D

Reply:
Die Frage ist aber auch, ob wir wirklich da nen beispiel brauchen, wenn wir unten ohnehin schon 7! Beispiele auflisten